README.md

```diff
@@ -173,8 +176,8 @@ A component the securely accepts an EBT PIN for balance requests and payment cap
 ```swift
 private let pinNumberTextField: ForagePINTextField = {
     let tf = ForagePINTextField()
-    tf.placeholder = "PIN Field"
-    tf.isSecureTextEntry = true
```

Comment:
PIN shouldn't have placeholder and we set/fix `isSecureTextEntry = true` internally